src/components/Calculator.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+
+export class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="calculator-flex">
+        <div className="output">
+          <div className="operand"> 0</div>
+        </div>
+
+      </div>
+    );
+  }
+}
+
+export default Calculator;
```

Comment:
- [ ] Nice work adding `operate.js` and `calculate.js` into the logic directory but Kindly import `calcuate.js` to the `Calculator` component and add all buttons inside it too to make it appear on the screen and use click events, state, and props  to make your app functional as expected because now there is only the output appears on the screen like the image below.

![image](https://user-images.githubusercontent.com/68796232/170702049-0bd8583a-90a7-4d34-9b25-8f571cfcae73.png)

Reply:
@Omar-Muhamad 
i made a mistake when push now I think its clear kindly check and confirm thank you